project/profiles/capacity4more/modules/c4m/user/c4m_user/actions/c4m_user_contact_multiple.action.inc

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/**
+ * @file
+ * Callback & forms related to the user contact multiple.
+ */
+
+/**
+ * Implements hook_action_info().
+ *
+ * A custom action to contact a multiple users with a mail action.
+ * Declares action for Site Admin, Group Admins and Group Owner.
+ */
+function c4m_user_contact_multiple_action_info() {
+  $group = c4m_og_current_group();
+  // If not within OG context, don't declare action.
+  if (!$group) {
+    return array();
+  }
+  // If not Site Admin, Group Admin or Group Owner, don't declare action.
+  if (!_c4m_features_og_members_is_power_user($group)) {
+    return array();
+  }
+
+  return array(
+    'c4m_user_contact_multiple_action' => array(
+      'type' => 'user',
+      'label' => t('Contact'),
+      'configurable' => TRUE,
+      'triggers' => array('any'),
+    ),
+  );
+}
+
+/**
+ * Action form function.
+ *
+ * Create 'Contact Multiple' form.
+ */
+function c4m_user_contact_multiple_action_form(array $context, $form_state) {
+  global $user;
+
+  // If 'Select all items on all pages' is selected.
+  if ($form_state['select_all_pages']) {
+    $group = c4m_og_current_group();
+    // Get ids of all active subscribers of current group.
+    $uids = og_get_group_members_properties($group, array(), 'members__' . OG_STATE_ACTIVE, 'node');
+  }
+  else {
+    // Get selected items on current page.
+    $uids = $form_state['selection'];
+  }
+
+  $accounts = user_load_multiple($uids);
+  $users_name = array();
+
+  // Create an array of users names for notified subscribers.
+  foreach ($accounts as $account) {
+    // Make sure that id is valid, and GO/GA didn't select himself.
+    if ($account->uid != 0 && $user->uid != $account->uid) {
```

Comment:
@ordavidil 
No, it's imposible.
uid != 0 check if to make sure that og_get_group_members_properties() doesn't return some account with invalid uid.
Should I remove it?